MapboxNavigation/NavigationViewController.swift

```diff
@@ -147,6 +147,31 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var shouldManageApplicationIdleTimer = true
     
+    /**
+     Enum denoting the types of the destination building highlighting on arrival.
+     */
+    public enum DestinationBuildingHighlightType: Int {
```

Comment:
Not critical, but there might be an appropriate category in the jazzy table of contents for this new symbol.

Reply:
Added `WaypointType` to `Maps` section in `jazzy.yml`.